src/Database/Expression/QueryExpression.php

```diff
@@ -393,6 +393,18 @@ public function count()
     }
 
     /**
+     * Build join condition with identifier wrapping.
```

Comment:
You right, it could be used in SET too. Do you have better method description in mind?